webui/react/src/components/WebhookCreateModal.tsx

```diff
@@ -50,11 +56,26 @@ const triggerOptions = [
     value: V1TriggerType.TASKLOG,
   },
 ];
+const modeOptions = [
+  {
+    label: 'All experiments in Workspace',
+    value: V1WebhookMode.WORKSPACE,
+  },
+  {
+    label: 'Specific experiment with matching configuration',
```

Comment:
q: can this mode ever match multiple experiments? if so maybe change copy to `Specific experiment(s) with...`

Reply:
@EmilyBonar Thanks for reviewing! 
Feel free to try out the feature, since the only changes here are related to creating webhooks, using webhooks should work as expected.